static/app/views/organizationIntegrations/integrationItem.tsx

```diff
@@ -50,7 +50,7 @@ const Labels = styled('div')<StyledProps>`
 `;
 
 const IntegrationName = styled('div')`
-  font-size: 1.6rem;
+  font-size: 1rem;
```

Comment:
Scaling down by a factor of 10/16 = 0.625.

<img width="251" alt="Screen Shot 2021-12-16 at 11 44 59 AM" src="https://user-images.githubusercontent.com/44172267/146445761-144b1bed-fc5b-48fb-af20-d8450d0748db.png">